WordPress/Classes/ViewRelated/Blog/QuickStartChecklistViewController.swift

```diff
@@ -62,6 +64,7 @@ class QuickStartChecklistViewController: UITableViewController {
                     self?.reload()
                     self?.tableView.scrollToRow(at: IndexPath(row: 0, section: 0), at: .top, animated: true)
                 }
+                WPAnalytics.track(.quickStartChecklistSkippedAll)
```

Comment:
Maybe bump this from the `skipAll` method provided the user doesn't decide to cancel the UIAlertController.  Here we're bumping everytime the button is tapped which I think is not what this event is about.  Thoughts?